cmd/smoke-test/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG ONOS_BUILD_VERSION=stable
```

Comment:
Just out of curiosity, why are you creating a docker image just for the test ? what does this have that we can't move to the base docker image ?